DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoSendBinding.cs

```diff
@@ -283,16 +274,28 @@ CancellationTokenSource cts
         converted = converter.ConvertToSpeckle(rhinoObject);
         converted.applicationId = applicationId;
       }*/
+      try
+      {
+        var converted = converter.Convert(rhinoObject);
+        converted.applicationId = applicationId;
 
-      var converted = converter.ConvertToSpeckle(rhinoObject);
-      converted.applicationId = applicationId;
-
-      // 4. add to host
-      collectionHost.elements.Add(converted);
-      _basicConnectorBinding.Commands.SetModelProgress(
-        modelCard.ModelCardId,
-        new ModelCardProgress { Status = "Converting", Progress = (double)++count / rhinoObjects.Count }
-      );
+        // 4. add to host
+        collectionHost.elements.Add(converted);
+        _basicConnectorBinding.Commands.SetModelProgress(
+          modelCard.ModelCardId,
+          new ModelCardProgress { Status = "Converting", Progress = (double)++count / rhinoObjects.Count }
+        );
+      }
+      catch (SpeckleConversionException e)
```

Comment:
yeah this is probably for some debate along with the reporting, exception handling pattern etc... I think it's fine as we evolve this. That's a POC ticket to "add a pattern for consideration", so all fine for now and reasonable.

Reply:
Agreed, this will most definitely need revision